packages/api-gateway-service/src/helpers.ts

```diff
@@ -2,10 +2,10 @@ const fetch = require( 'node-fetch' );
 
 /* This consists of the list of microservices which needs to be part of the gateway */
 export const serviceList = [
-  { name: 'User Service', url: `http://${ process.env.USER_SERVICE_SERVICE_HOST }/graphql` },
-  { name: 'Home Service', url: `http://${ process.env.HOME_SERVICE_SERVICE_HOST }/graphql` },
-  { name: 'Feedback Service', url: `http://${ process.env.FEEDBACK_SERVICE_SERVICE_HOST }/graphql` },
-  { name: 'Notification Service', url: `http://${ process.env.NOTIFICATION_SERVICE_SERVICE_HOST }/graphql` },
+  { name: 'User Service', url: `http://${ process.env.USER_SERVICE_SERVICE_HOST }:8080/graphql` },
+  { name: 'Home Service', url: `http://${ process.env.HOME_SERVICE_SERVICE_HOST }:8080/graphql` },
+  { name: 'Feedback Service', url: `http://${ process.env.FEEDBACK_SERVICE_SERVICE_HOST }:8080/graphql` },
+  { name: 'Notification Service', url: `http://${ process.env.NOTIFICATION_SERVICE_SERVICE_HOST }:8080/graphql` },
```

Comment:
Why do we need to explicitly hardcode the port for the services?

Reply:
@deshmukhmayur told that microservices are running on port 8080 and for the purpose of reusing the environment variables in openshift

Reply:
@sayak-sarkar That's the only port we'll be using it on. And the only port exposed within the local environment of the cluster. So I didn't see any problem with it.
If we had to change it, we can make use of the environment variable `USER_SERVICE_SERVICE_PORT `.
But I don't think that's necessary though.